src/tpm2_wrap.c

```diff
@@ -3980,6 +3996,61 @@ int wolfTPM2_CreateAndLoadAIK(WOLFTPM2_DEV* dev, WOLFTPM2_KEY* aikKey,
     return rc;
 }
 
+int wolfTPM2_CreateKeySeal(WOLFTPM2_DEV* dev, WOLFTPM2_KEYBLOB* keyBlob,
+    WOLFTPM2_HANDLE* parent, TPMT_PUBLIC* publicTemplate,
+    const byte* auth, int authSz, const byte* sealData, int sealSize)
```

Comment:
In my tests the maximum size of sealSize was 128 bytes `MAX_SYM_DATA`. Please add some comments about this in examples/README.md and improve the error response on seal command.